ts/fields/FieldsPage.svelte

```diff
@@ -1,8 +1,212 @@
 <!--
-jopyright: Ankitects Pty Ltd and contributors
+Copyright: Ankitects Pty Ltd and contributors
 License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
 -->
 <script lang="ts">
+    import { writable } from "svelte/store";
+
+    import IconConstrain from "../components/IconConstrain.svelte";
+    import { altPressed, controlPressed, shiftPressed } from "../lib/keys";
+    import { PaneArray } from "../lib/pane-array";
+    import { Notetypes, notetypes } from "../lib/proto";
+    import { randomUUID } from "../lib/uuid";
+    import type { DrawerItemType } from "./drawer";
+    import { Drawer } from "./drawer";
+    import DrawerTitle from "./DrawerTitle.svelte";
+    import FieldConfigEditor from "./FieldConfigEditor.svelte";
+    import { fieldIcon } from "./icons";
+    import InsertionPoint from "./InsertionPoint.svelte";
+    import { Panes } from "./panes";
+
+    const ntid = 1641830698585;
```

Comment:
Heads-up for those who want to try out the PR: this notetype id is hardcoded! You need to exchange it with one that exists on the current profile.